suites/squid/rbd/tier-3_rbd_migration.yaml

```diff
@@ -90,3 +90,26 @@ tests:
         fio:
           size: 100M
       polarion-id: CEPH-83584070
+
+  - test:
+      desc: Live migration of RAW format images with encryption
+      config:
+        encryption_type:
+          - luks1
+          - luks2
+        rep_pool_config:
+          num_pools: 1
+          do_not_create_image: true
```

Comment:
why are we avoiding creating images here?

Reply:
this is handled in the script as we are creating one image for each encryption format (luk1,luks2)